pkg/statistics/handle/syncload/stats_syncload_test.go

```diff
@@ -318,5 +318,18 @@ func TestRetry(t *testing.T) {
 		t.Logf("task1.ResultCh should get nothing")
 		t.FailNow()
 	}
+	task1.Retry = 0
+	for i := 0; i < syncload.RetryCount*5; i++ {
+		task1, err1 = h.HandleOneTask(testKit.Session().(sessionctx.Context), task1, exitCh)
+		require.Error(t, err1)
```

Comment:
assert the err1 msg

Reply:
It failpoints a panic in the program. so ```HandleOneTask``` can quickly exit and create many goroutine in the tests.